### PR TITLE
Update Jenkinsfile

### PR DESCRIPTION
**Pull request by kltm**
Tagging

### CODE REVIEW

**Comment by kltm** [comment]
Tagging @dougli1sqrd

File: Jenkinsfile [hunk 1]
```diff
@@ -493,6 +493,9 @@ pipeline {
 			    // There are other tacks we might take
 			    sh 'python3 ./mypyenv/bin/pip3 install -r requirements.txt'
 			    sh 'python3 ./mypyenv/bin/pip3 install ../graphstore/rule-runner'
+			    // WARNING: Temporary patch code to prevent bad pandas/gaferencer situation.
+			    python3 ./mypyenv/bin/pip3 uninstall -y pandas
+			    python3 ./mypyenv/bin/pip3 install pandas==0.24.2
```
Comment (dougli1sqrd) [inline Jenkinsfile]: We need to wrap each command in `sh`, right?

**Comment by kltm** [comment]
@dougli1sqrd updated

**Comment by kltm** [comment]
@okayed from @dougli1sqrd on gitter

**Pull request merged**